Dockerfile

```diff
@@ -38,6 +38,8 @@ LABEL license='SPDX-License-Identifier: Apache-2.0' \
   copyright='Copyright (c) 2023: Intel'
 
 RUN apk add --update --no-cache dumb-init
+# Ensure using latest versions of all installed packages to avoid any recent CVEs
+RUN apk --no-cache upgrade
```

Comment:
would it make sense to combine these two lines to avoid creating an extra layer?

Reply:
You mean something like:
```
RUN apk add --update --no-cache dumb-init;  RUN apk --no-cache upgrade
```
Yes, that could be an future optimization.